source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -149,24 +149,22 @@ export default class WalletSendForm extends Component<Props, State> {
         label: this.context.intl.formatMessage(messages.receiverLabel),
         placeholder: this.context.intl.formatMessage(messages.receiverHint),
         value: '',
-        validators: [({ field, form }) => {
+        validators: [async ({ field, form }) => {
```

Comment:
@DominikGuzei seems like these changes cause a glitch in the send form :(
During the validation in case you enter invalid (e.g. too big amount) the `total amount` get's shown briefly and then it is removed once the validation fails. We should not show the `total amount` until the validation finishes.

**Animated GIF of the issue**
![send-form-glitch](https://user-images.githubusercontent.com/376611/39083752-29557360-456a-11e8-8782-c437adfd2afb.gif)